components/phpunit_bridge.rst

```diff
@@ -19,10 +20,13 @@ It comes with the following features:
 
 * Displays the stack trace of a deprecation on-demand;
 
-* Provides a ``ClockMock`` and ``DnsMock`` helper classes for time or network-sensitive tests.
+* Provides a ``ClockMock``, ``DnsMock`` and ``ClassExistsMock`` classes for tests
+  sensitive to time, network or class existence.
 
-* Provides a modified version of PHPUnit that does not embed ``symfony/yaml`` nor
-  ``prophecy`` to prevent any conflicts with these dependencies.
+* Provides a modified version of PHPUnit that allows 1. separating the
+  dependencies of your app from those of phpunit to prevent any unwanted
+  constraints to apply; 2. running tests in parallel when a test suite is split
```

Comment:
Maybe this, parallelizing tests, could be highlighted more? Do you think it's worth adding a small section to this page giving a brief introduction why it's useful and how it works? If so, I could make a PR when I have some time.

Reply:
It would be nice indeed, I was totally unaware of that one...

Reply:
@dbrumann this has been merged, but it'd be nice to add a short section about parallelizing tests. Thanks.

Reply:
Yes, I will do an additional PR maybe this weekend.